package project;

import java.sql.SQLException;

public class Main {
	public static void main(String args[]) {
		
	   /* try {
			Reservation_Of_Halls_Tables res = new Reservation_Of_Halls_Tables();
		} catch (SQLException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}*/
		/*try {
			Example ex = new Example();
		} catch (SQLException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}*/
		/*try {
			products pro = new products();
		} catch (SQLException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}*/
		
		/*try {
			OrderingFood food = new OrderingFood();
		} catch (SQLException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}*/

		//RegForm rg = new RegForm();
		//Login_Registration bd = new Login_Registration();
		
	  Client_Staff cl_st = new Client_Staff();
		
		/*try {
			Information_About_Reservation_Of_Halls_Tables info = new Information_About_Reservation_Of_Halls_Tables();
		} catch (SQLException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}*/
		
		/*try {
			Information_About_Ordering_Food info2 = new Information_About_Ordering_Food();
		} catch (SQLException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}*/
		

		// Login log = new Login();

	}
}
